components/brave_shields/resources/panel/components/main-panel/index.tsx

```diff
@@ -135,6 +135,7 @@ function MainPanel () {
               isOn={siteBlockInfo?.isShieldsEnabled}
               onChange={handleToggleChange}
               accessibleLabel={getLocale('braveShieldsEnable')}
+              disabled={siteBlockInfo?.isShieldsManaged}
```

Comment:
how is this actually displaying to the user? Is it obvious that it's disabled by policy? What happens if the user tries to click anyway? We need browser tests to check that the UI shows the correct value and behaves correctly when the setting is managed.

Reply:
you can see the picture in the description 
![image](https://user-images.githubusercontent.com/2965009/197778871-cc326c7c-2f97-458c-80cf-dc8447d6ba14.png)